source/whats-new.txt

```diff
@@ -12,12 +12,34 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.4 <version-4.4>`
 * :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.4:
+
+What's New in 4.4
+-----------------
+
+New features of the 4.4 Node.js driver release include:
+
+- KMIP support when using CSFLE.
```

Comment:
Issue: KMIP to "KMIP provider". [Context](https://jira.mongodb.org/browse/DOCSP-20564)

```suggestion
- KMIP provider support when using CSFLE.
```

Reply:
I believe provider is implicit here...